pytorch_lightning/trainer/properties.py

```diff
@@ -49,8 +49,7 @@ class TrainerProperties(ABC):
     _default_root_dir: str
     _lightning_optimizers = None
     _progress_bar_callback: ProgressBarBase
-    _running_stage: Optional[RunningStage] = None
-    _state: TrainerState
+    state: TrainerState = None
```

Comment:
nit: TrainerState = None will give mypy errors right? 
do we need the default value?

Reply:
Removed it